pkg/cli/cli.go

```diff
@@ -491,15 +492,20 @@ func cvdCommands(opts *subCommandOpts) []*cobra.Command {
 	// Local image
 	create.Flags().BoolVar(&createFlags.LocalImage, localImageFlag, false,
 		"Builds a CVD with image files built locally, the required files are https://cs.android.com/android/platform/superproject/+/master:device/google/cuttlefish/required_images and cvd-host-packages.tar.gz")
-	localImgMutuallyExFlags := []string{
+	// Local artifacts flag
+	create.Flags().StringVar(&createFlags.LocalArtifacts, localArtifacsFlag, "",
+		"Path to file containing the local artifacts sources")
```

Comment:
I think it's better to have more detailed information about what `local artifacts sources` means, such as `toml file containing paths of local artifacts`.